src/components/ToastCard/ToastCard.tsx

```diff
@@ -49,10 +54,7 @@ export default function ToastCard({ toastInstance, type, text, undo }: Props) {
     >
       <div className="toast-card__body">
         {iconName && <i className={`p-icon--${iconName}`}>Success</i>}
-        <div
-          className="toast-card__message"
-          dangerouslySetInnerHTML={{ __html: text }}
-        ></div>
+        <div className="toast-card__message">{children}</div>
```

Comment:
I couldn't see a good reason for this to use `dangerouslySetInnerHTML` instead of just being passed JSX.